insolar/payload/payload.go

```diff
@@ -139,6 +141,10 @@ func (h *MessageHash) IsZero() bool {
 	return true
 }
 
+func (m RequestInfo) RequestReference() *insolar.Reference {
```

Comment:
Same here, let's move this somewhere else.

Reply:
Done; I've rewritten code, but forgot to remove old wrapper.